.gitignore

```diff
@@ -30,3 +30,13 @@ dist/
 # Local Netlify folder
 .netlify
 src/emulator/release/
+
+# Language Server Protocol cache
+.lsp/
+.lsp/.cache/
+.lsp/.cache/db.transit.json
```

Comment:
These lines belong in the global .gitignore on your local machine, not in our repo